routes/user.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const router = express.Router();
+const crypt = require('bcryptjs')
+const restricted = require('../data/restricted-middleware');
+
+const db = require('../data/dbConfig.js');
+const Users = require('../data/users_model.js')
+
+const errors = { 
+  '19':'Another entree has the same value no duplicates'
+}
+
+router.post('/register',(req,res)=>{
+  let user = req.body;
```

Comment:
Might be a good idea to include some conditionals in case the user doesn't input a username or password, or uses an incorrect field name (like "username" or "pass").